### ISSUE TITLE
70 unit tests failed in CI, on a trivial change. Looks like Provider issues. 

### ISSUE THREAD

**Issue by trentmc** [oceanprotocol/ocean.py#1374]
### The bug

I had created [PR#1373](https://github.com/oceanprotocol/ocean.py/pull/1373) to fix [#1369](https://github.com/oceanprotocol/ocean.py/pull/1373), wrt setup-remote.md instructions.

It was just changes to READMEs/setup-remote.md.

Yet when CI automatically ran, 70 unit tests failed. See below. [Full log](https://github.com/oceanprotocol/ocean.py/actions/runs/4467282355/jobs/7846538921).

And, if I re-run, they fail again. And I re-run again, they fail again. Argh.

It looks like Provider.

Critical priority because we can't proceed with ocean.py until this is fixed. (Likely needs a fix in Provider? So, when Provider is fixed and then ocean.py CI passes, we can close this.)

```text
=========================== short test summary info ============================
FAILED ocean_lib/aquarius/test/test_aquarius.py::test_aqua_functions_for_single_ddo
FAILED ocean_lib/assets/test/test_asset_downloader.py::test_ocean_assets_download_destination_file
FAILED ocean_lib/data_provider/test/test_data_service_provider.py::test_initialize_fails
FAILED ocean_lib/data_provider/test/test_data_service_provider.py::test_start_compute_job_fails_empty
FAILED ocean_lib/data_provider/test/test_data_service_provider.py::test_delete_job_result
FAILED ocean_lib/data_provider/test/test_data_service_provider.py::test_encrypt
FAILED ocean_lib/data_provider/test/test_data_service_provider.py::test_fileinfo_and_initialize
FAILED ocean_lib/data_provider/test/test_data_service_provider.py::test_c2d_environments
FAILED ocean_lib/data_provider/test/test_data_service_provider.py::test_provider_address
FAILED ocean_lib/data_provider/test/test_data_service_provider.py::test_provider_address_with_url
FAILED ocean_lib/data_provider/test/test_data_service_provider.py::test_get_root_uri
FAILED ocean_lib/data_provider/test/test_data_service_provider.py::test_build_endpoint
FAILED ocean_lib/data_provider/test/test_data_service_provider.py::test_build_specific_endpoints
FAILED ocean_lib/data_provider/test/test_data_service_provider.py::test_check_single_file_info
FAILED ocean_lib/data_provider/test/test_data_service_provider.py::test_encrypt_failure
FAILED ocean_lib/data_provider/test/test_data_service_provider.py::test_fileinfo_failure
FAILED ocean_lib/data_provider/test/test_data_service_provider.py::test_initialize_failure
FAILED ocean_lib/data_provider/test/test_data_service_provider.py::test_initialize_compute_failure
FAILED ocean_lib/data_provider/test/test_data_service_provider.py::test_job_result_failure
FAILED ocean_lib/data_provider/test/test_data_service_provider.py::test_check_asset_failure
FAILED ocean_lib/ocean/test/test_ocean_assets.py::test_update_metadata - requ...
FAILED ocean_lib/ocean/test/test_ocean_assets.py::test_update_credentials - r...
FAILED ocean_lib/ocean/test/test_ocean_assets.py::test_update_datatokens - re...
FAILED ocean_lib/ocean/test/test_ocean_assets.py::test_update_flags - request...
FAILED ocean_lib/ocean/test/test_ocean_assets.py::test_ocean_assets_search - ...
FAILED ocean_lib/ocean/test/test_ocean_assets.py::test_ocean_assets_algorithm
FAILED ocean_lib/ocean/test/test_ocean_assets.py::test_create_url_asset - req...
FAILED ocean_lib/ocean/test/test_ocean_assets.py::test_plain_asset_with_one_datatoken
FAILED ocean_lib/ocean/test/test_ocean_assets.py::test_plain_asset_multiple_datatokens
FAILED ocean_lib/ocean/test/test_ocean_assets.py::test_plain_asset_multiple_services
FAILED ocean_lib/ocean/test/test_ocean_assets.py::test_encrypted_asset - requ...
FAILED ocean_lib/ocean/test/test_ocean_assets.py::test_compressed_asset - req...
FAILED ocean_lib/ocean/test/test_ocean_assets.py::test_compressed_and_encrypted_asset
FAILED ocean_lib/ocean/test/test_ocean_assets.py::test_create_algo_asset - re...
FAILED ocean_lib/ocean/test/test_ocean_assets.py::test_create_pricing_schemas[2]
FAILED ocean_lib/ocean/test/test_ocean_assets.py::test_create_pricing_schemas[1]
FAILED tests/flows/test_reuse_order_fees.py::test_reuse_order_fees[Ocean-7]
FAILED tests/flows/test_reuse_order_fees.py::test_reuse_order_fees[MockDAI-7]
FAILED tests/flows/test_reuse_order_fees.py::test_reuse_order_fees[MockUSDC-7]
FAILED tests/flows/test_reuse_order_fees.py::test_reuse_order_fees[Ocean-0]
FAILED tests/flows/test_reuse_order_fees.py::test_reuse_order_fees[MockUSDC-0]
FAILED tests/flows/test_reuse_order_fees.py::test_reuse_order_fees[Ocean-0.000000000000000001]
FAILED tests/flows/test_reuse_order_fees.py::test_reuse_order_fees[MockUSDC-0.000001]
FAILED tests/flows/test_reuse_order_fees.py::test_reuse_order_fees[Ocean-700]
FAILED tests/flows/test_reuse_order_fees.py::test_reuse_order_fees[MockUSDC-700]
FAILED tests/flows/test_start_order_fees.py::test_start_order_fees[Ocean-5-6-7]
FAILED tests/flows/test_start_order_fees.py::test_start_order_fees[MockDAI-5-6-7]
FAILED tests/flows/test_start_order_fees.py::test_start_order_fees[MockUSDC-5-6-7]
FAILED tests/flows/test_start_order_fees.py::test_start_order_fees[Ocean-0-0-0]
FAILED tests/flows/test_start_order_fees.py::test_start_order_fees[MockUSDC-0-0-0]
FAILED tests/flows/test_start_order_fees.py::test_start_order_fees[Ocean-0.000000000000000001-0.000000000000000001-0.000000000000000001]
FAILED tests/flows/test_start_order_fees.py::test_start_order_fees[MockUSDC-0.000001-0.000001-0.000001]
FAILED tests/flows/test_start_order_fees.py::test_start_order_fees[Ocean-500-600-700]
FAILED tests/flows/test_start_order_fees.py::test_start_order_fees[MockUSDC-500-600-700]
FAILED tests/integration/ganache/test_consume_flow.py::test_consume_asset[simple]
FAILED tests/integration/ganache/test_consume_flow.py::test_consume_asset[graphql]
FAILED tests/integration/ganache/test_consume_flow.py::test_consume_asset[onchain]
FAILED tests/integration/ganache/test_consume_flow.py::test_consume_asset[arweave]
FAILED tests/integration/ganache/test_disconnecting_components.py::test_with_wrong_provider
FAILED tests/integration/ganache/test_graphql.py::test_consume_simple_graphql_query
FAILED tests/integration/ganache/test_graphql.py::test_consume_parametrized_graphql_query
FAILED tests/integration/ganache/test_market_flow.py::test_market_flow[publisher]
FAILED tests/integration/ganache/test_market_flow.py::test_market_flow[another_user]
FAILED tests/integration/ganache/test_market_flow.py::test_pay_for_access_service_good_default
FAILED tests/integration/ganache/test_onchain.py::test_consume_parametrized_onchain_data
FAILED tests/readmes/test_readmes.py::test_script_execution[script0] - reques...
FAILED tests/readmes/test_readmes.py::test_script_execution[script2] - reques...
FAILED tests/readmes/test_readmes.py::test_script_execution[script4] - reques...
FAILED tests/readmes/test_readmes.py::test_script_execution[script7] - reques...
FAILED tests/readmes/test_readmes.py::test_script_execution[script9] - reques...
ERROR tests/integration/ganache/test_compute_flow.py::test_compute_raw_algo
ERROR tests/integration/ganache/test_compute_flow.py::test_compute_registered_algo
ERROR tests/integration/ganache/test_compute_flow.py::test_compute_reuse_order
ERROR tests/integration/ganache/test_compute_flow.py::test_compute_multi_inputs
ERROR tests/integration/ganache/test_compute_flow.py::test_compute_trusted_algorithm
ERROR tests/integration/ganache/test_compute_flow.py::test_compute_update_trusted_algorithm
ERROR tests/integration/ganache/test_compute_flow.py::test_compute_trusted_publisher
ERROR tests/integration/ganache/test_compute_flow.py::test_compute_just_provider_fees
```

**Comment by mariacarmina** [comment]
#1265 merged

**Comment by trentmc** [comment]
This was fixed via merging multichain provider work to main. https://github.com/oceanprotocol/ocean.py/commit/dc45be3412015c283f8f0f3fbd7ed29129e0f354

Then main with multichain was merged to [PR#1373](https://github.com/oceanprotocol/ocean.py/pull/1373). CI tests passed!

Therefore this issue can be closed too:)
